### PR TITLE
wip enable guest wicked logs

### PR DESCRIPTION
**Pull request by guoxuguang**
[log enhancement] collect and upload guest wicked debugging logs

- Verification run: 
[gi-guest_developing-on-host_developing-kvm](http://149.44.176.58/tests/5367218#)

### CODE REVIEW

File: tests/virt_autotest/libvirt_virtual_network_init.pm [hunk 1]
```diff
@@ -85,10 +85,19 @@ sub run_test {
         virt_autotest::utils::ssh_copy_id($guest);
         #Prepare the new guest network interface files for libvirt virtual network
         assert_script_run("ssh root\@$guest 'cd /etc/sysconfig/network/; cp ifcfg-eth0 ifcfg-eth1; cp ifcfg-eth0 ifcfg-eth2; cp ifcfg-eth0 ifcfg-eth3; cp ifcfg-eth0 ifcfg-eth4; cp ifcfg-eth0 ifcfg-eth5; cp ifcfg-eth0 ifcfg-eth6'");
+        #enable guest wickedd debugging
+        #assert_script_run "ssh root\@$guest 'perl -i -lpe 's{^(WICKED_DEBUG)=.*}{\$1=\"all\"};s{^(WICKED_LOG_LEVEL)=.*}{\$1=\"debug\"}' /etc/sysconfig/network/config'";
+        #assert_script_run "ssh root\@$guest \"egrep \"WICKED_LOG_LEVEL\" /etc/sysconfig/network/config\"";
```
Comment (okurz) [inline tests/virt_autotest/libvirt_virtual_network_init.pm:90]: Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

**Review by okurz — COMMENTED** [review]
The commit message could have a bit more details, e.g. what issue is fixed, why this needs to change, to which versions of which product it applies, link to a bug or a fate entry, the choices you made, etc. Also see https://commit.style/ or http://chris.beams.io/posts/git-commit/ as a helpful guide how to write good commit messages.

And s/wickded/wicked/ :)

File: tests/virt_autotest/libvirt_virtual_network_init.pm (hunk 1 above)
Comment (guoxuguang) [reply tests/virt_autotest/libvirt_virtual_network_init.pm:90]: Cool! had updated now. 
Thanks so much

**Comment by guoxuguang** [comment]
> The commit message could have a bit more details, e.g. what issue is fixed, why this needs to change, to which versions of which product it applies, link to a bug or a fate entry, the choices you made, etc. Also see https://commit.style/ or http://chris.beams.io/posts/git-commit/ as a helpful guide how to write good commit messages.
> 
> And s/wickded/wicked/ :)

Cool, got it. 
Also, had added more details here.

**Pull request closed without merging**